services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Consider using `Optional[List[int]]` for `genres_ids` and `actors_ids` to indicate that these parameters can be `None`. This requires importing `List` and `Optional` from `typing`.